PdfJsViewerPlugin.inc.php

```diff
@@ -67,17 +68,27 @@ function articleCallback($hookName, $args) {
 		$templateMgr = TemplateManager::getManager($request);
 		if ($galley && $galley->getFileType() == 'application/pdf') {
 			$application = Application::getApplication();
+
+      $journal = $request->getJournal();            
+   
+      $pdfUrl = $journal->getPath() . '/article/download/' . $article->getBestArticleId() . '/' . $galley->getBestGalleyId();
+   
+      if (Config::getVar('general', 'restful_urls') != 'Off') {
+        $pdfUrl = '/index.php/' . $pdfUrl;
+      } 
+
 			$templateMgr->assign(array(
-				'displayTemplateResource' => $this->getTemplateResource('display.tpl'),
+				'pluginTemplatePath' => $this->getTemplatePath(),
 				'pluginUrl' => $request->getBaseUrl() . '/' . $this->getPluginPath(),
 				'galleyFile' => $galley->getFile(),
 				'issue' => $issue,
 				'article' => $article,
 				'galley' => $galley,
 				'jQueryUrl' => $this->_getJQueryUrl($request),
 				'currentVersionString' => $application->getCurrentVersion()->getVersionString(false),
+        'pdfUrl' => $pdfUrl
 			));
-			$templateMgr->display($this->getTemplateResource('articleGalley.tpl'));
+			$templateMgr->display($this->getTemplatePath() . '/articleGalley.tpl');
```

Comment:
I think this reverts a necessary change in the `master` branch in preparation for OJS 3.2. Maybe you're working with the `ojs-stable-3_1_1` branch of this plugin?